src/SignalR/server/Core/src/HubResult.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public struct HubResult
+    {
+        public bool MethodInvoked;
+        public object Result;
+
+        public static HubResult WithResult(object result)
+        {
+            return new HubResult { MethodInvoked = true, Result = result };
+        }
+
+        public static HubResult NotInvoked()
```

Comment:
nit: Naming? What does MethodInvoked really mean? What is the user trying to articulate?

Reply:
> Naming?

Definitely.

> What is the user trying to articulate?

The user is trying to say that the hub method was rejected.